### PR TITLE
New entry form

### PR DESCRIPTION
**Pull request by BrennaManning**

### CODE REVIEW

**Comment by aconsilvio** [comment]
YAY THE NEW FORM AND THE NEW DATABASE WORKS AND IT'S PRETTY

**Comment by davidabrahams** [comment]
OMG YOU GUYS ARE THE BESSSSTTTTT!!!! Out of curiosity, what was the issue?

**Comment by davidabrahams** [comment]
Checked mLab. It's there!!!

File: public/views/content/newEntryForm.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+              </label>
+              <label class="radio-inline">
+                <input ng-model="newEntry.period" type="radio" name="optradio" value="Long Term">Long Term</label>
+            </div>
+          </div>
+
+          <div class="form-group">
+            <label for="sel1">Attendence:</label>
+            <select class="form-control" id="sel1" ng-model="newEntry.attendence">
+              <option value="Present">Present</option>
```
Comment (shrutiyer) [inline public/views/content/newEntryForm.html:50]: sel1? Is it possible to name it something that describes it better?

**Pull request merged**